### PR TITLE
feat(PDP): Variants-testcases

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**
@kibo-kevinwatts @kibo-sushant @kibo-chandradeeptalaha @amolpawar-kibo

### CODE REVIEW

File: pages/product/index.vue [hunk 1]
```diff
@@ -1,313 +1,344 @@
...
+                    :has-show-password="false"
+                  />
+                </div>
+              </div>
+
+              <SfDivider class="divider-first" />
+
+              <KiboFulfillmentOptions
+                :fulfillment-options="fulfillmentOptions"
+                selcted="store"
```
Comment (kibo-kevinwatts) [inline pages/product/index.vue]: fix type, selected :)

File: pages/product/index.vue [hunk 2]
```diff
@@ -1,313 +1,344 @@
...
+
+    const addToCart = () => {
+      console.log("Add to Cart qunatity: ", qtySelected.value)
+    }
+    const addToWishList = () => {
+      console.log("Add to Whislist qunatity: ", qtySelected.value)
+    }
+
+    // Fullfillment Options
+    const fulfillmentOptions = useState(`pdp-fulfillment-options`, () => [
```
Comment (kibo-kevinwatts) [inline pages/product/index.vue]: Get fulfillmentTypesSupported in GQL query
`query { product { fulfillmentTypesSupported } }`

File: pages/product/index.vue (hunk 1 above)
Comment (kibo-sushant) [reply pages/product/index.vue]: Resolved

File: pages/product/index.vue (hunk 2 above)
Comment (kibo-sushant) [reply pages/product/index.vue]: Resolved

**Review by kibo-chandradeeptalaha — APPROVED** [review]

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**